packages/sql/README.md

```diff
@@ -8,13 +8,14 @@ A SQL toolkit for Effect.
 import { Config, Effect, Struct, pipe } from "effect"
 import * as Sql from "@effect/sql"
 import * as Pg from "@effect/sql-pg"
+import { SqlClient } from "@effect/sql/SqlClient"
```

Comment:
```suggestion
import { SqlClient } from "@effect/sql"
import { PgClient } from "@effect/sql-pg"
```

This is the recommended import style.

Reply:
Yep, can do. Thanks for the quick feedback.

Reply:
Shall I get rid of the odd inline comment here? https://github.com/Effect-TS/effect/blob/main/packages/sql/README.md?plain=1#L106